clang/tools/symbol-scanner/BinarySymbolScanner.py

```diff
@@ -54,6 +54,61 @@
 import sys
 import subprocess
 from pathlib import Path
+import pandas as p
+import numpy as np
+# Class which creates the dictionary of API Migration Support Status
+class api_mapping_support:
+    def __init__(self):
+        # Array of URLs for the API Mapping CSV files in SYCLomatic repo
+        self.api_mapping_urls=["https://raw.githubusercontent.com/oneapi-src/SYCLomatic/SYCLomatic/docs/dev_guide/api-mapping-status/CUB_API_migration_status.csv",
```

Comment:
1.  ASM_API_migration_status.csv and wmma_API_migration_status.csv  are not used.
2.  Have introduced "Partial" keywords in the *csv file, which means not all kind usage of the API is migration supported.